terminusdb_client/woqlquery/woql_query.py

```diff
@@ -482,7 +482,7 @@ def _find_last_subject(self, json):
     def _compile_path_pattern(self, pat):
         """Turns a textual path pattern into a JSON-LD description"""
         toks = _tokenize(pat)
-        if toks and len(toks):
+        if toks and toks:
```

Comment:
since now it's "toks and toks", we can just check "if toks"

Reply:
yeah right. updating the PR.

Reply:
@Cheukting updated the PR